skiko/src/commonTest/kotlin/org/jetbrains/skia/StrutStyleTests.kt

```diff
@@ -16,6 +17,10 @@ class StrutStyleTests {
             strutStyle.fontStyle = FontStyle(300, 4, FontSlant.ITALIC)
 
             assertEquals(FontStyle(300, 4, FontSlant.ITALIC), strutStyle.fontStyle)
+
+
+            strutStyle.setFontFamilies(arrayOf("MonacoShmonaco"))
```

Comment:
Is it actual font name?

Reply:
it's not but in the framework of validation of setting fonts this test is sufficient (we test actual font loading in some different tests)